internal/scorecard/alpha/config.go

```diff
@@ -36,12 +37,15 @@ type Config struct {
 }
 
 // LoadConfig will find and return the scorecard config, the config file
-// can be passed in via command line flag or from a bundle location or
-// bundle image
-func LoadConfig(configFilePath string) (Config, error) {
+// is found from a bundle location (TODO bundle image)
+// scorecard config.yaml is expected to be in the bundle at the following
+// location:  tests/scorecard/config.yaml
+func LoadConfig(bundlePath string) (Config, error) {
 	c := Config{}
 
-	// TODO handle getting config from bundle (ondisk or image)
+	configFilePath := filepath.Join(bundlePath, "tests", "scorecard", "config.yaml")
+
+	// TODO handle bundle images, not just on-disk
```

Comment:
I actually think a better option might be to leave this as is and to have the CLI use the bundle path by default. If `--config` is passed on the command line, that would override the default. WDYT?

Reply:
ok, I added --config back in, but made the bundle location the default.